platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/WrappedPersistenceManager.java

```diff
@@ -36,6 +36,10 @@
     this.persistenceManager = persistenceManager;
   }
 
+  PersistenceManager getInnerPersistenceManager() {
```

Comment:
❓ Should this be protected since it is meant for subclasses? 
If it is package-private because of testing, then annotate with @VisibleForTesting